dhcpv6/modifiers_test.go

```diff
@@ -95,3 +95,13 @@ func TestWithDomainSearchList(t *testing.T) {
 	require.Equal(t, "slackware.it", labels[0])
 	require.Equal(t, "dhcp.slackware.it", labels[1])
 }
+
+func TestWithFQDN(t *testing.T) {
+	var m Message
+	WithFQDN(4, "cnos.localhost")(&d)
+	require.Equal(t, 1, len(d.Options))
+	ofqdn := d.Options[0].(*OptFQDN)
```

Comment:
s/d/m

Reply:
Done

Clearly I was defining my message and eye copying other functions in same file.  Changed per s/m/d, and also consistent with other functions in this file.